nixos/modules/services/games/minecraft-server.nix

```diff
@@ -158,6 +158,32 @@ in {
           + "-XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10";
         description = "JVM options for the Minecraft server.";
       };
+
+      fifo = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "/run/mincraft/stdin";
```

Comment:
I'd say let's just always enable this, there's no harm to it. So this means removing the `null` case and specifying this example as a default.